src/main/java/com/alibaba/com/caucho/hessian/io/BasicSerializer.java

```diff
@@ -106,6 +106,8 @@ public void writeObject(Object obj, AbstractHessianOutput out)
                 break;
 
             case FLOAT:
+                out.writeDouble(Double.parseDouble(((Number) obj).floatValue() + ""));
```

Comment:
What is the purpose of  `+ ""`?

Reply:
My mistake, I have modified it, please review it.